src/readii/io/loaders/features.py

```
@@ -0,0 +1,74 @@
+        # Get the full path to the feature file
+        feature_file_path = os.path.join(extracted_feature_dir, image_type_feature_file)
+            
+        # Load the feature data into a pandas dataframe
+        raw_feature_data = loadFileToDataFrame(feature_file_path)
+
+        try:
+            # Drop the labels from the dataframe if specified
+            if drop_labels:
+                # Data is now only extracted features
+                raw_feature_data.drop(labels_to_drop, axis=1, inplace=True)
+        except Exception as e:
```

Ruff (0.8.0)

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)